pkg/properties/properties.go

```diff
@@ -32,3 +32,5 @@ const FakeCluster = "fake_cluster"
 
 // nolint:gosec // Linter thinks there are hardcoded credentials here...
 const UseLocalCredentials = "use_local_credentials"
+
+const ProvisionShardId = "provision_shard_id"
```

Comment:
This property is for developer scenarios, right?
I mean, do we expect that users require to add the provision_shard_id ?

Reply:
it is optional (providing properties is an hidden flag). This PR doesn't change anything if we are not providing something like
--properties provision_shard_id:xxx in input